packages/swapper/src/swappers/PortalsSwapper/getPortalsTradeQuote/getPortalsTradeQuote.ts

```diff
@@ -132,18 +131,21 @@ export async function getPortalsTradeQuote(
       const dummyInputToken = `${portalsNetwork}:${dummySellAssetAddress}`
       const dummyOutputToken = `${portalsNetwork}:${dummyBuyAssetAddress}`
 
-      const maybeGasLimit = await fetchPortalsTradeOrder({
+      const dummyOrderResponse = await fetchPortalsTradeOrder({
         sender: dummyQuoteParams.accountAddress,
         inputToken: dummyInputToken,
         outputToken: dummyOutputToken,
         inputAmount: dummyQuoteParams.sellAmountCryptoBaseUnit,
-        slippageTolerancePercentage: 10, // hardcoded high slippage tolerance for the dummy quote, the actual one will use the correct one
+        slippageTolerancePercentage: maybeSlippageTolerancePercentageOverride,
         partner: getTreasuryAddressFromChainId(sellAsset.chainId),
         feePercentage: affiliateBpsPercentage,
         validate: true,
         swapperConfig,
       })
-        .then(({ context }) => context.gasLimit)
+        .then(({ context }) => ({
+          maybeGasLimit: context.gasLimit,
+          autoSlippageTolerancePercentage: context.slippageTolerancePercentage,
```

Comment:
Gut feel is this would yield unpredictable results: gas dummy quote is fine as most ERC-20s are pretty much the same except for a few with more logic, but slippage will vary greatly depending on the pair since the dummy quote is using a different token/amount.
q: does this consistently yield a *higher* silppage? if so, that's actually perf!

Reply:
You are right to call this out - I'm not super happy with it so I've reverted the slippage behaviour for non-valid quotes (where we need to use a dummy quote) in https://github.com/shapeshift/web/pull/7654/commits/1c47819ccdfc81feef30b22305aaaf2ff33f861b to use the default.

I have a follow-up planned using the `estimate` endpoint that'll better address this need, though don't want to muddy this PR.